apps/web/components/chat/message-input.tsx

```
@@ -386,14 +386,14 @@ export function MessageInput({ channelName, draft, replyTo, onCancelReply, onSen
         >
           <Reply className="w-3 h-3 -scale-x-100" style={{ color: "var(--theme-text-muted)" }} />
           <span style={{ color: "var(--theme-text-muted)" }}>Replying to</span>
-          <span className="font-semibold text-white">
+          <span className="font-semibold" style={{ color: "var(--theme-text-bright)" }}>
             {replyTo.author?.display_name || replyTo.author?.username}
           </span>
           <span className="truncate flex-1" style={{ color: "var(--theme-text-muted)" }}>
             {replyTo.content}
           </span>
           <button onClick={onCancelReply} aria-label="Cancel reply" className="focus-ring rounded" style={{ color: "var(--theme-text-muted)" }}>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)